config/app_dynamics_agent.yml

```diff
@@ -17,6 +17,7 @@
 ---
 version: 4.1.+
 repository_root: ! '{default.repository.root}/app-dynamics'
-default_application_name:
-default_node_name: ! '$(expr "$VCAP_APPLICATION" : ''.*instance_index[": ]*\([[:digit:]]*\).*'')'
+default_application_name: 
+#default_node_name: ! '$(expr "$VCAP_APPLICATION" : ''.*instance_index[": ]*\([[:digit:]]*\).*'')'
+default_node_name: ! '$(expr "$VCAP_APPLICATION" : '.*"application_name[": ]*\\([a-z_0-9-]\\+\\).*')[$(expr "$VCAP_APPLICATION" : '.*"instance_index[": ]*\\([0-9]\\+\\).*')]-[provo]'
```

Comment:
Someone may want to double check that expr syntax.  My regex skills are passable at best.